### ISSUE TITLE
100+ Fields gets cut off

### ISSUE THREAD

**Issue by Midknight** [focuslabllc/craft-field-cheat-sheet#39]
![craftcmscheatsheet](https://cloud.githubusercontent.com/assets/274881/19934674/1f195c96-a0dd-11e6-9bf7-6d4a9602051b.png)

I have 111 fields and it cuts off after 69 Fields + 9 Field Groups.

**Comment by erikreagan** [comment]
@Midknight Hey there. Apologies for the very delayed response. I'm trying to track this down now but am having trouble replicating it. Can you share other details about your experience such as the browser, version, etc?

**Comment by Midknight** [comment]
Yeah no problem. Running Firefox 50, Chrome 55.0.2883.87 (Chrome cuts list off at 15 items as well within a group). I am assuming its something to do with your max-height on

`aside ul li input[type="checkbox"]:checked ~ ul`

Reason for multiple fields is that I have dedicated product pages that have numerous models with the same specifications per product. Little easier to manipulate and pull data when individual specs can be called versus table. Also keeps my data entry people in check.

**Comment by erikreagan** [comment]
Thanks for that info. I tried both Firefox 50.x and Chrome 55.x and had 133 fields in the test build. Some groups had 15+ fields in them.

Any other details you can share? Any interest in sending me a db dump so I can duplicate more about the environment?

**Comment by Midknight** [comment]
I just played with the max-height on: `aside ul li input[type="checkbox"]:checked ~ ul` and changed it from 30em to 100% and it resolved my issue.
